iac/tf-multienv-cicd-anthos-autopilot/shell-scripts/00-setup-all-the-things.sh

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 Google LLC
```

Comment:
I wonder if another name / something less ambiguous would be more suitable, like `00-run-scripts-01-through-05` or ``00-all-scripts` or something...

Reply:
As a proposal I have removed all bash scripts and instead added step-by-step instructions for the initial (one-time) setup to the README.md